app/models/concerns/waste_carriers_engine/can_have_registration_attributes.rb

```diff
@@ -95,7 +95,13 @@ def charity?
         business_type == "charity"
       end
 
+      def uk_location?
+        %w[england scotland wales northern_ireland].include?(location)
```

Comment:
Shall we make those constants in some relevant place?

Reply:
I definitely think we should do this – however I started working on it and it grew in scope pretty rapidly! So I may tackle it in a separate PR if that is OK with you?